widget/balance/src/main/java/com/ivy/widget/balance/WalletBalanceWidgetContent.kt

```diff
@@ -201,3 +201,8 @@ fun ButtonsSection(
         }
     }
 }
+
+// Sort categories alphabetically
+fun sortCategoriesAlphabetically(categories: List<String>): List<String> {
```

Comment:
Where this function is called? It seems like this change does nothing but just creating a new function that sorts strings